tox.ini

```diff
@@ -0,0 +1,12 @@
+[tox]
+envlist =
+    py{27,33,34,35,36,37}
```

Comment:
I disagree to some extent. Might not be worth testing all of them. But for example python 34 introduced enums and py35 introduced os.walk which have broken compatibility for me in the past.